lib/index.js

```diff
@@ -7,6 +7,9 @@ const styles = `
 .current {
   fill: #5fa04e;
 }
+.commercial {
+  fill: #eb6d27;
```

Comment:
```suggestion
  fill: #ae5f00;
```

Contrast ratio pass and it `danger-600` on nodejs.org design system